src/pages/index.svelte

```diff
@@ -73,19 +76,133 @@
+      />
+      <Card
+        languageAbbreviation="FR"
+        src="/images/workshops/creer-ton-propre-itineraire-conseils-pratiques-outils.png"
+        date="10 June 2021"
+        time="19:00 - 20:30"
+        title="Créer ton propre itinéraire: conseils pratiques & outils"
+        href="https://propreitineraire.eventbrite.be/?aff=Website"
+      />
+      <Card
+        languageAbbreviation="EN"
+        src="/images/workshops/create-your-own-itinerary-practical-tips-tools.png"
+        date="15 June 2021"
+        time="19:00 - 20:30"
+        title="Create your own itinerary: practical tips & tools"
+        href="https://ownitinerary.eventbrite.be/?aff=Website"
+      />
+    </div>
+
+    <div class="stmf-column">
+      <div class="stmf-column-header"><h4>{$_('stmf.sections.contribute')}</h4></div>
+      <Card
+        group="Beginner"
+        languageAbbreviation="EN"
+        src="/images/workshops/collect-data-and-improve-maps-while-slow-travelling-introduction-to-openstreetmap.png"
+        date="17 June 2021"
+        time="19:00 - 20:30"
+        title="Collect data and improve maps while slow travelling: Introduction to OpenStreetMap"
+        href="https://openstreetmap-beginner.eventbrite.be/?aff=Website"
+      />
+      <Card
+        group="Advanced"
+        languageAbbreviation="EN"
+        src="/images/workshops/collect-data-and-improve-maps-while-slow-travelling-introduction-to-openstreetmap.png"
+        date="22 June 2021"
+        time="19:00 - 20:30"
+        title="Collect data and improve maps while slow travelling: Introduction to OpenStreetMap"
+        href="https://openstreetmap-advanced.eventbrite.be/?aff=Website"
+      />
+      <Card
+        languageAbbreviation="EN"
+        src="/images/workshops/opportunities-of-worlds-biggest-encyclopedia-for-slow-travellers-wikipedia.png"
+        date="24 June 2021"
+        time="19:00 - 20:30"
+        title="Opportunities of world's biggest encyclopedia for slow travellers: Wikipedia"
+        href="https://wikipedia-workshop.eventbrite.be/?aff=Website"
+      />
+    </div>
+
+    <div class="stmf-column">
+      <div class="stmf-column-header"><h4>{$_('stmf.sections.rethink')}</h4></div>
+      <Card
+        languageAbbreviation="EN"
+        src="/images/workshops/slow-travelling--the-commons-the-role-of-the-welcome-to-my-garden-community.png"
+        date="29 June 2021"
+        time="19:00 - 20:30"
+        title="Slow travelling & the commons: the role of the Welcome To My Garden community"
+        href="https://slowtravel-commons.eventbrite.be/?aff=Website"
+      />
+    </div>
+  </div>
+</section>
+
+<section class="getInspired">
+  <GetInspiredFlyer
+    section="plan"
+    title="Create your own itinerary: practical tips &amp; tools"
+    languageAbbreviation="EN"
+    date="15 June 2021"
+    time="19:00 - 20:30"
+    sponsor1={groteroutepadenLogo}
+    sponsor2={lesSentiersDeGrandeRandonneeLogo}
```

Comment:
Would prefer this to be an array, but we're not going to reuse this so it's okay